src/main/java/cz/muni/fi/ComponentType.java

```diff
@@ -0,0 +1,8 @@
+package cz.muni.fi;
+
+/**
+ * Author: Richard Hrmo
+ */
+public enum ComponentType {
+    ENGINE, TRANSMISSION, BRAKES, AERO, BODY
```

Comment:
zmen prosim AERO na AERODYNAMIC a BODY na SUSPENSION, nech sa to zhoduje s Car class